me.jtheoof.swappy.yaml

```diff
@@ -0,0 +1,32 @@
+id: me.jtheoof.swappy
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: swappy
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+rename-desktop-file: swappy.desktop
+rename-icon: swappy
+modules:
+  - name: swappy
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+      - -Dman-pages=disabled
+    post-install:
+      - install -Dm644 ../$FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/jtheoof/swappy
+        tag: v1.5.1
+        commit: a996b2cec65aadd04ded780f141b02c5bdfbeb32
+        disable-shallow-clone: true
```

Comment:
Why?

Reply:
In the meson.build, `git` is used to create the `--version` string.

https://github.com/jtheoof/swappy/blob/596b9a8b17234303a9d4d2ea1d3107cb5442253e/meson.build#L17-L18

But after testing it, I can see that the commit hash is the same.